iotdb-core/datanode/src/main/java/org/apache/iotdb/db/consensus/DataRegionConsensusImpl.java

```diff
@@ -132,6 +139,39 @@ private static ConsensusConfig buildConsensusConfig() {
                               CONF.getRegionMigrationSpeedLimitBytesPerSecond())
                           .build())
                   .build())
+          .setPipeConsensusConfig(
+              PipeConsensusConfig.newBuilder()
+                  .setRPC(
+                      PipeConsensusConfig.RPC
+                          .newBuilder()
+                          .setConnectionTimeoutInMs(CONF.getConnectionTimeoutInMS())
+                          .setRpcSelectorThreadNum(CONF.getRpcSelectorThreadCount())
+                          .setRpcMinConcurrentClientNum(CONF.getRpcMinConcurrentClientNum())
+                          .setRpcMaxConcurrentClientNum(CONF.getRpcMaxConcurrentClientNum())
+                          .setIsRpcThriftCompressionEnabled(CONF.isRpcThriftCompressionEnable())
+                          .setSelectorNumOfClientManager(CONF.getSelectorNumOfClientManager())
+                          .setThriftServerAwaitTimeForStopService(
+                              CONF.getThriftServerAwaitTimeForStopService())
+                          .setThriftMaxFrameSize(CONF.getThriftMaxFrameSize())
+                          .setMaxClientNumForEachNode(CONF.getMaxClientNumForEachNode())
```

Comment:
Be sure to check if these configurations is suitable for your connector algorithm, for example, by default there will be 300 concurrent client per receiver node....